lib/firepad.js

```diff
@@ -16,12 +17,13 @@ firepad.Firepad = (function(global) {
   var LIST_TYPE = firepad.LineFormatting.LIST_TYPE;
   var CodeMirror = global.CodeMirror;
   var ace = global.ace;
+  var monaco = global.monaco;
 
   function Firepad(ref, place, options) {
     if (!(this instanceof Firepad)) { return new Firepad(ref, place, options); }
 
-    if (!CodeMirror && !ace) {
-      throw new Error('Couldn\'t find CodeMirror or ACE.  Did you forget to include codemirror.js or ace.js?');
+    if (!CodeMirror && !ace && !global.monaco) {
+      throw new Error('Couldn\'t find CodeMirror, ACE or Monaco.  Did you forget to include codemirror.js/ace.js/monaco.d.ts?');
```

Comment:
Hrm.  I thought a .d.ts file only has type definitions and so it won't help to include it, right (it's not going to create global variables and whatnot).  Can you fix this to be more correct, or just remove it?

Reply:
I tried to remove this, but it is giving some error.

Reply:
To be clear, I just meant remove "/monaco.d.ts" from the error message...